packages/core/src/renderer/Equation.ts

```diff
@@ -40,11 +40,27 @@ const RenderEquation = (
     // If equations are rendered as plain TeX strings, forward relevant props to a <text> element and surround the TeX string with $$
     // Since the `svg` TeX package render text with the center on the baseline, we shift the labels down by height/2 + descent
     const baselineY = y + shape.height.contents / 2 - shape.descent.contents;
-    const txt = placeholderString(
+
+    let txt = placeholderString(
       `$${getAdValueAsString(shape.string)}$`,
       [x, baselineY],
       shape
     );
+
+    // If the Equation has a texContourColor passthrough value, give the
+    // string a contour with the specified color
```

Comment:
I'm a bit unsure whether `texContourColor` should be a passthrough value, since this field and value appear to be something that Penrose directly handles.

I would suggest perhaps modifying the definition of `Equation` to include this field. [This file](https://github.com/penrose/penrose/blob/6b2550eee5afdcc3e106bdb41e129bead6a51947/packages/core/src/shapes/Equation.ts#L15) is the definition of `Equation`. [Here ](https://github.com/penrose/penrose/blob/6b2550eee5afdcc3e106bdb41e129bead6a51947/packages/core/src/compiler/shapeChecker/CheckShape.ts#LL143C31-L143C31) is the Shape Typechecker which would also need to be adapted to include the new field.

Reply:
I'm going to leave this one alone, because the whole "export to TeX" thing is experimental at this point, and this is also connected to #1420.  Long term the right solution is just to restory the `strokeColor` field to `Equation`, and then use this field to trigger a `\contour` command on TeX export.  I started going down this road, but it requires quite a few changes to the shape definitions and renderer.  So, better to put that in a different PR than to make this one even more complex.